client/src/components/CompetitionHome/CompetitionHome.jsx

```diff
@@ -18,13 +18,44 @@ import { wcaUrl } from "../../lib/urls";
 import { flatMap } from "../../lib/utils";
 import { competitionCountries } from "../../lib/competition";
 import { getTimezone } from "../../lib/date";
+import RecordList from "../RecordList/RecordList";
 
 const COMPETITION_QUERY = gql`
   query Competition($id: ID!) {
     competition(id: $id) {
       id
       wcaId
       name
+      competitionRecords {
+        id
+        tag
```

Comment:
Let's create `components/RecordList/fragments.jsx` with `RECORD_LIST_RECORD_FRAGMENT`, so that we can share it in both places.

Existing example:

https://github.com/thewca/wca-live/blob/f11c7634e9778f8bd4098908a138f90800541669/client/src/components/admin/AdminRound/fragments.jsx#L5-L6